microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,71 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-operators-olm"]
+= Using Operator Lifecycle Manager with {microshift-short}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-operators-olm
+
+toc::[]
+
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You must add your own catalogs for the Operators you want to use with your applications.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available.
+* Install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/cli_tools/opm-cli#cli-opm-install[`opm` CLI]
+
+[IMPORTANT]
+====
+Before using an Operator, verify with the provider whether the Operator is supported on {product-title}.
+====
+
+[id="microshift-installing-olm-options_{context}"]
+== Determining your installation type for OLM
+There are different ways to install OLM with {microshift-short}, depending on your use case.
+
+* To use OLM with {microshift-short}, you can install the OLM RPM along with the {microshift-short} RPM on {op-system-base-full}.
+//future xref to come once install doc is complete
+* You can also embed OLM in a {op-system-ostree-first} image. When embedding in a {op-system-ostree} image, use the installation path that matches your intended use case: either with a network connection, or without one.
+//** See future xref to come once install doc is complete
+//** See for a disconnected installation (xref to come once doc complete)
+
+[id="microshift-options-adding-operator-catalogs_{context}"]
+== Options for adding custom Operator catalogs
+You can add custom Operator catalogs that you can manage with OLM in one of two ways:
+
+* If you know the exact few Operators you want to use, you can use file-based Operator catalogs, see the example catalog in the "File-based Operator catalogs" section.
```

Comment:
I think the beginning of these two bullets is too vaguely similar to me.

> If you know the exact few Operators you want to use, you can use file-based Operator catalogs

The File-based Catalog use case is more about "building a custom catalog potentially with your in-house operators and not filtering an existing catalog image".  "know the exact few Operators you want to use" sounds like filtering to me.

Looking at the existing OLM doc for a custom catalog:
https://docs.openshift.com/container-platform/4.14/operators/admin/olm-managing-custom-catalogs.html

> "Cluster administrators and Operator catalog maintainers can create and manage custom catalogs packaged using the bundle format on Operator Lifecycle Manager (OLM) in OpenShift Container Platform."

I think we can highlight the "custom catalog" as the OLM doc is doing.